dls-controls/malcolmjs
# Feature/better test coverage

## Description

Improved unit test coverage for widgets and reducers

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm test
- view coverage on codecov

## Agile board tracking

connect to #196
